test/Microsoft.AspNetCore.JsonPatch.Test/IntegrationTests/AnonymousObjectIntegrationTest.cs

```diff
@@ -0,0 +1,242 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using Microsoft.AspNetCore.JsonPatch.Exceptions;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNetCore.JsonPatch.Test.IntegrationTests
+{
+    public class AnonymousObjectIntegrationTest
+    {
+        [Fact]
+        public void ShouldAddToList_WithDifferentCase()
+        {
+            var targetObject = new
+            {
+                IntegerList = new List<int>() { 1, 2, 3 }
+            };
+
+            // create patch
```

Comment:
Comments like this aren't really needed they are just obvious

Reply:
I also realize that this is just the copy-paste part so I'm going to stop commenting